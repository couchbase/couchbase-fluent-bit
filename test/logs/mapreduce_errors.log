[mapreduce_errors:error,2021-09-29T11:18:39.517Z,couchdb_ns_1@cb.local:<0.18870.1>:couch_set_view_updater:-do_maps/3-fun-0-:885]Bucket `travel-sample`, main group `_design/dev_test`, error mapping document `<ud>test</ud>` for view `repro`: <ud>too long key emitted: 4806 bytes (line 2:2)</ud>
