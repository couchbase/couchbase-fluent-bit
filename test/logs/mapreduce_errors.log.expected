/fluent-bit/test/logs/mapreduce_errors.log.actual: [1632914319.517000000, {"file_path":"/fluent-bit/test/logs","file":"mapreduce_errors.log","filename":"/fluent-bit/test/logs/mapreduce_errors.log","logger":"mapreduce_errors","timestamp":"2021-09-29T11:18:39.517Z","message":"couchdb_ns_1@cb.local:<0.18870.1>:couch_set_view_updater:-do_maps/3-fun-0-:885]Bucket `travel-sample`, main group `_design/dev_test`, error mapping document `<ud>test</ud>` for view `repro`: <ud>too long key emitted: 4806 bytes (line 2:2)</ud>","pod":{"name":"test-pod","namespace":"default"},"hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","couchbase":{"cluster":"test-cluster","node":"cb-node.test-cluster.default.svc","logging.version":"99"},"level":"ERROR","level_numeric":"40"}]
